### PR TITLE
Update prometheus to v2.10.0 and grafana to v6.2.5

### PR DESCRIPTION
**Pull request by MiguelNdeCarvalho**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+read USERNAME 
+echo 'Enter the desired Password for the configuration'
+read PASSWORD 
+
+export ADMIN_USER $USERNAME            #Set the username
+export ADMIN_PASSWORD $PASSWORD        #Set the password
+
+cd /tmp
+
+git clone https://github.com/MiguelNdeCarvalho/dockprom.git
```
Comment (stefanprodan) [inline setup.sh]: Please change to git URL to point at this repo

**Comment by nightah** [comment]
I don't personally see any real value in the `setup.sh` file, firstly you need to clone the repo to have the setup file to be able to run it.

Second why would you want to clone inside /tmp and then run it? It just seems redundant.

Happy to merge this in if you drop the `setup.sh`, I've just tested and confirmed that the newer versions do not have any impact to the existing stack.

**Comment by stefanprodan** [comment]
@nightah you are right, I've updated my comment.

**Comment by MiguelNdeCarvalho** [comment]
It was only suposse to do a pull request with the first commit only lol

**Pull request closed without merging**